surrealdb/clients/ws.py

```diff
@@ -58,7 +59,7 @@ def __init__(
         self._ws: ClientWebSocketResponse
         self._recv_task: asyncio.Task
 
-        self._responses: dict[str, RPCResponse] = {}
+        self._response_futures: dict[str, asyncio.Future] = {}
```

Comment:
In the imports, include
`from typing import Dict`

```suggestion
        self._response_futures: Dict[str, asyncio.Future] = {}
```

Reply:
oops